docs/generators/java-helidon.md

```diff
@@ -0,0 +1,304 @@
+---
```

Comment:
Is this file added manually?

Files under docs/generators are supposed to be auto-generated.

Reply:
It is auto-generated. I added the generated file to git manually, as with new or updated generated samples file. I guess that's incorrect? I'll remove it from the PR then.

Reply:
@wing328 Can you or someone please trigger the workflow again after my latest push? Thanks.